packages/datadog-instrumentations/src/couchbase.js

```diff
@@ -217,15 +217,15 @@ addHook({ name: 'couchbase', file: 'lib/cluster.js', versions: ['^2.6.5'] }, Clu
 
 // semver >=3 <3.2.0
 
-addHook({ name: 'couchbase', file: 'lib/collection.js', versions: ['>=3.0.0 <3.2.0'] }, Collection => {
+addHook({ name: 'couchbase', file: 'lib/collection.js', versions: ['^3.0.7', '^3.1.3'] }, Collection => {
```

Comment:
`^3.0.7` is the same as `>=3.0.7 <4.0.0` so that would already include the range of `^3.1.3`.